catalog/plugin/catalog-plugin-defaultsecurityattributevalues/src/main/java/org/codice/alliance/catalog/plugin/defaultsecurity/DefaultSecurityAttributeValuesPlugin.java

```diff
@@ -23,165 +25,183 @@
-import org.opensaml.core.xml.schema.XSString;
-import org.opensaml.saml.saml2.core.AttributeStatement;
+import org.codice.ddf.security.SystemHighAttributes;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+/**
+ * This plugin sets security attributes on {@link Metacard}s to system high attribute values if none
+ * of the policies were able to apply security markings to this {@link Metacard}.
+ *
+ * <p>The configuration for this plugin defines the mapping for 6 security {@link Attribute} names
+ * to system high attribute names.
+ *
+ * <p>Each one of the 6 configuration keys corresponds to a security {@link Attribute} name:
+ *
+ * <table>
+ * <tr>
+ *     <td> Configuration key </td>                                       <td> {@link Attribute} name </td>
+ * </tr>
+ * <tr>
+ *     <td> {@value CLASSIFICATION_CONFIGURATION_KEY} </td>               <td> {@value Security#CLASSIFICATION} </td>
+ * </tr>
+ * <tr>
+ *     <td> {@value RELEASABILITY_CONFIGURATION_KEY} </td>                <td> {@value Security#RELEASABILITY} </td>
+ * </tr>
+ * <tr>
+ *     <td> {@value CODEWORDS_CONFIGURATION_KEY} </td>                    <td> {@value Security#CODEWORDS} </td>
+ * </tr>
+ * <tr>
+ *     <td> {@value DISSEMINATION_CONTROLS_CONFIGURATION_KEY} </td>       <td> {@value Security#DISSEMINATION_CONTROLS} </td>
+ * </tr>
+ * <tr>
+ *     <td> {@value OTHER_DISSEMINATION_CONTROLS_CONFIGURATION_KEY} </td> <td> {@value Security#OTHER_DISSEMINATION_CONTROLS} </td>
+ * </tr>
+ * <tr>
+ *     <td> {@value OWNER_PRODUCER_CONFIGURATION_KEY} </td>               <td> {@value Security#OWNER_PRODUCER} </td>
+ * </tr>
+ * </table>
+ *
+ * <p>For example, say the value for the key={@value CLASSIFICATION_CONFIGURATION_KEY} in the
+ * configuration is "Clearance" when a {@link Metacard} is processed by this plugin. The value for
+ * the system high attribute named "Clearance" is "U". The {@link Attribute} named {@value
+ * Security#CLASSIFICATION} will be set to "U" on that {@link Metacard}.
+ *
+ * <p>If the system high attribute does not exist, the corresponding security {@link Attribute} will
+ * not be set on any processed {@link Metacard}s, and processing will not be interrupted.
+ */
 public class DefaultSecurityAttributeValuesPlugin implements PreIngestPlugin {
 
-  private final SecurityAttributes securityAttributes;
-
-  private final Supplier<Subject> subjectSupplier;
+  private static final Logger LOGGER =
+      LoggerFactory.getLogger(DefaultSecurityAttributeValuesPlugin.class);
 
-  private Map<String, Attribute> securityMarkings;
+  private static final String CLASSIFICATION_CONFIGURATION_KEY = "classification";
 
-  public static final String DEFAULTMARKINGS = "defaultMarkings";
+  private static final String RELEASABILITY_CONFIGURATION_KEY = "releasability";
 
-  static final String CLASSIFICATION_KEY = "classification";
+  private static final String CODEWORDS_CONFIGURATION_KEY = "codewords";
```

Comment:
I thought we were going to make this sciControls?

Reply:
`"codewords"` is just the key in the properties when the Plugin in updated, not the default value:
_blueprint.xml_
```xml
        <argument>
            <map>
               ...
                <entry key="codewords" value="sciControls"/>
                ...
            </map>
        </argument>
```
_metatype.xml_
```xml
		<AD name="Codewords" id="codewords"
			required="true"
			type="String" 
			default="sciControls"
			description="Mappings of system attributes to Codewords metacard marking.">
		</AD>
```